tests/Unit/Impl/StreamxClientIntegrationTest.php

```diff
@@ -99,6 +104,59 @@ private function shouldPublishAndUnpublishDataMessage(string $key, $dataPayload)
         $this->assertDataIsUnpublished($key);
     }
 
+    //** @test */
```

Comment:
Note: since we are on a public repo - this integration test (that needs a running StreamX instance) is by default disabled

Reply:
can this be done in other way as commenting and uncommenting? the test should not be started on the build but shoult be startable by command plus mesh which should be started for this test should be in codebase and comand to run mesh and test should be in readme. I guess you have this anyway prepared. 
In case of making it hard to excude the test during build and starting it from some command just keep current appraoch with commenting